<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Skills</class>
 <widget class="QDialog" name="Skills">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="defSkillOne">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>410</y>
     <width>80</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="defSkillTwo">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>290</y>
     <width>80</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="defSkillThree">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>176</y>
     <width>80</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="offSkillOne">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>404</y>
     <width>80</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="offSkillTwo">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>284</y>
     <width>80</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="offSkillThree">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>80</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
